assets/components/src/sortable-newsletter-list-control/style.scss

```diff
@@ -0,0 +1,46 @@
+.newspack__newsletter-list-control {
+	&__selected {
+		.newspack-card {
+			display: flex;
+			// flex-direction: column;
```

Comment:
Commented out code should be removed.

Reply:
Fixed in 90e05a5a4c8edf4d38951ef1391d6ac278cb4f2e